health/health.go

```diff
@@ -7,22 +7,39 @@ import (
 
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
-	healthpb "google.golang.org/grpc/health/grpc_health"
+	"google.golang.org/grpc/codes"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1alpha"
 )
 
 // HealthCheck is the client side function to health-check a server
-func HealthCheck(t time.Duration, cc *grpc.ClientConn) error {
+func HealthCheck(t time.Duration, cc *grpc.ClientConn, service_name string) (*healthpb.HealthCheckResponse, error) {
```

Comment:
this convenience func seems not add much value. Let's remove it and let users make a call by themselves.